test/test_op_aliases.py

```diff
@@ -128,6 +128,22 @@ def __init__(self,
               lambda d: torch.randn(20, device=d),
               get_args=lambda d: (torch.randn(20, device=d),),
               decorators=(onlyCPU,)),
+    # NOTE: only runs on CPU because it leaks CUDA memory
+    #   (see https://github.com/pytorch/pytorch/issues/43119)
+    AliasInfo('divide', torch.divide, 'div', torch.div,
+              lambda d: torch.randn(20, device=d), get_args=lambda d: (torch.rand(20, device=d) + .1,),
+              decorators=(onlyCPU,)),
```

Comment:
so we check CUDA memory leaks but not CPU?  Or it doesn't leak on CPU?

Reply:
We don't check CPU memory leaks. This will be fixed when the tests are refactored.